sw/airborne/arch/chibios/subsystems/actuators/actuators_uavcan.c

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright (C) 2013 AggieAir, A Remote Sensing Unmanned Aerial System for Scientific Applications
+ * Utah State University, http://aggieair.usu.edu/
+ *
+ * Michal Podhradsky (michal.podhradsky@aggiemail.usu.edu)
```

Comment:
I suspect the copyright can be updated as we were not involved into making this file:-)

Reply:
Will change that ^^